src/main/java/tech/gruppone/stalker/server/exceptions/InvalidEmailException.java

```diff
@@ -0,0 +1,12 @@
+package tech.gruppone.stalker.server.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@SuppressWarnings("serial")
+@ResponseStatus(HttpStatus.BAD_REQUEST)
+public class InvalidEmailException extends RuntimeException {
+  public InvalidEmailException() {
+    super("The given email address is not valid for this request.");
```

Comment:
"Valid" di solito intende che una cosa rispetta uno schema, quindi una cosa o è "valid" o non lo è, non può essere "valid for this, but invalid for that". "Acceptable" forse ci sta meglio.

Reply:
in questo caso specifico in realtà mi sono accorto che l amaggior parte dei servizi non ti dice neanche se l'email non è valida (e.g. quando non esiste nel db) perché la cosa dà "superficie di attacco" nel senso che puoi dedurre dall'esito se quell'email è registrata o meno al srvizio. quindi rimuovo proprio sta roba sia da api che da sorgente java

Reply:
In effetti è anche quello vhe dice Cardin